src/util.ts

```diff
@@ -69,28 +92,88 @@ const configDescriptors: ConfigDescriptor[] = [
     arg: "silent",
     isPath: false,
   },
+  {
+    name: "fallback",
+    env: "FALLBACK",
+    file: "fallback",
+    arg: "fallback",
+    isPath: false,
+  },
+  {
+    name: "registryPort",
+    env: "REGISTRY_PORT",
+    file: "registryPort",
+    arg: "registryPort",
+    isPath: false,
+  },
 ];
 
+/**
+ * Configuration object for a Local Tableland instance.
+ */
 export type Config = {
+  /**
+   * Instance of a Tableland Validator. If docker flag is set, this must be the
+   * full repository.
+   */
   validator?: string;
+
+  /**
+   * IPath to the Tableland Validator directory.
+   */
   validatorDir?: string;
+
+  /**
+   * Instance of a Tableland Registry.
+   */
   registry?: string;
+
+  /**
+   * Instance of a Tableland Registry.
+   */
   registryDir?: string;
+
+  /**
+   * Path to the Tableland Registry contract repository.
+   */
   docker?: boolean;
+
+  /**
+   * Use Docker to run the Validator.
+   */
   verbose?: boolean;
+
+  /**
+   * Silence all output to stdout.
+   */
   silent?: boolean;
+
+  /**
+   * Use a fallback Registry port if the default port 8545 is in use or a custom
+   * port is desired. Note that clients will need to be configured to use this
+   * port over the default port, e.g., connect to `http://127.0.0.1:<fallbackPort>`
+   * instead of port 8545.
+   */
+  fallback?: boolean;
```

Comment:
Does this work from a practical standpoint?  The chain will start on the fallback port, but how will the Validator know how to connect to the fallback?

For the sake of getting the CI tests to pass it seems easier to just wait until the standard port is available.  If we discover that can't work, we could create a non-standard validator config and retry starting with that and the `registryPort` options set.